cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+  // prompting user for 3 numbers to add 1/2
+  short num1;
+  cout<<"please enter first number: "<<endl;
+  cin>>num1;
+  
+  short num2;
+  cout<<"please enter second number: "<<endl;
+  cin>>num2; 
+  
+  short num3;
+  cout<<"please enter third number: "<<endl;
+  cin>>num3;
```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add